### DISCUSSION TITLE
Mantine is truly special

### DISCUSSION THREAD

**Discussion by arminnaimi** [mantinedev/mantine#6244] [Kind words]
Recently we introduced Mantine at work and there have been a few hiccups with using Mantine and Tailwind together, but it mostly has been a smooth transition and the devs are already so productive. It really feels like having an additional developer on the team. Kudos to the Mantine contributors for building something truly special.

I also have been wanting to try out Radix Themes and it is quite nice and it falls under the same category of what I like to call "Component Systems", but Mantine is just miles ahead and the conveniences that Mantine has built-in are just so thoughtful. As I am using Radix Themes, I wish I had Mantine 🤭

**Comment by rtivital** [comment]
Thanks for kind words!
